AK Zeus/app/src/main/java/ayds/zeus/songinfo/moredetails/fulllogic/DataBase.kt

```diff
@@ -0,0 +1,71 @@
+package ayds.zeus.songinfo.moredetails.fulllogic
+
+import android.content.ContentValues
+import android.content.Context
+import android.database.Cursor
+import android.database.sqlite.SQLiteDatabase
+import android.database.sqlite.SQLiteOpenHelper
+
+const val ARTIST_TABLE = "artists"
+const val ID_COLUMN = "id"
+const val ARTIST_COLUMN = "artist"
+const val INFO_COLUMN = "info"
+const val SOURCE_COLUMN = "source"
+const val URL_COLUMN = "url"
+const val SELECTION = "$ARTIST_COLUMN  = ?"
+const val SORT_ORDER = "$ARTIST_COLUMN DESC"
+
+class DataBase(context: Context?) : SQLiteOpenHelper(context, "dictionary.db", null, 1) {
+    override fun onCreate(db: SQLiteDatabase) {
+        db.execSQL("create table $ARTIST_TABLE ($ID_COLUMN INTEGER PRIMARY KEY AUTOINCREMENT, $ARTIST_COLUMN string, $INFO_COLUMN string, $SOURCE_COLUMN integer, $URL_COLUMN string)")
+    }
+
+    override fun onUpgrade(db: SQLiteDatabase, oldVersion: Int, newVersion: Int){}
+
+    companion object {
+        fun saveArtist(dbHelper: DataBase, artist: String, info: String,url:String) {
+            val db = dbHelper.writableDatabase
+            val values = ContentValues()
+            values.put(ARTIST_COLUMN, artist)
```

Comment:
pueden usar la facilidad de koltin 
```suggestion
            values.apply {
                put()
                put()
                ...
            }
```